### ISSUE TITLE
Write off difference amount

### ISSUE THREAD

**Issue by barredterra** [alyf-de/banking#106]
## Received a cent too much

- Bank transaction: € 42.42
- Unpaid invoice: € 42.41

Current behavior: Bank Transaction remains unreconciled with € 0.01

Desired behavior: € 0.01 is added in **Payment Entry**'s _Deductions and Loss_ (profit, in this case), **Bank Transaction** is fully reconciled.

## Received a cent too little

- Bank transaction: € 42.41
- Unpaid invoice: € 42.42

Current behavior: **Sales Invoice** remains Partially Paid/Overdue with € 0.01

Desired behavior: the total outstanding amount is allocated to the **Sales Invoice**, the missing € 0.01 is added in **Payment Entry**'s _Deductions and Loss_.

## UX idea

If the _To Allocate_ amount in the _Match vouchers_ tab is not zero and _Unpaid Vouchers_ is enabled, show a checkbox "Write off € 0.01". If that is checked, offer a Link field where the user can select the appropriate account to be used in _Deductions and Loss_.  _To Allocate_ goes to zero.

**Comment by Nudelsalad** [comment]
Hey @barredterra,

I think this issue is closely related to my issue, #270. Both address the need to write off small differences during payment reconciliation.

The main difference is that my use case involves amounts larger than a single cent. Could the solution for this be designed to handle a configurable tolerance amount?

That would solve both issues at once, and we could close #270. Thanks!

**Comment by barredterra** [comment]
Yes, the 1 cent in this example is just for illustrative purposes. It should work for whatever amount is left over.
